tools/tpm_keystore/cert_encrypt_tpm.c

```diff
@@ -0,0 +1,503 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information.
+ *
+ * Copyright (c) 2021 Kalycito Infotech Private Limited
+ */
+
+/* gcc cert_encrypt_tpm.c -o cert_encrypt_tpm -ltpm2_pkcs11 -lssl -lcrypto */
+
+#include <stdio.h>
```

Comment:
Add this to the tools build.
And activate in the CI.

Give this a —help option that explains the command line options.